src/components/hr/hr.stories.mdx

```diff
@@ -17,18 +17,13 @@ import StyledSystemProps from '../../../.storybook/utils/styled-system-props';
 Provides a horizontal dividing line to make the above and below components feel distinctly separate from one another.
 
 ## Quick Start
-
 To use Hr, import the `Hr` and set the margins to align it with other components.
 
 ```jsx
 import Hr from "carbon-react/lib/components/hr";
-
-const MyComponent = () => (
-  <Hr />
-);
 ```
 
-## Visuals
+## Examples
```

Comment:
The stickyfooter for one of the `Forms` in the following examples is rendering at the bottom of the page 
![Screenshot 2021-03-05 at 14 24 07](https://user-images.githubusercontent.com/35449967/110128225-77acbe80-7dbe-11eb-88b8-69947066dd3b.png)

Reply:
Its a feature ;)

Reply:
This one is in master, not sure how to fix it

Reply:
Have fixed it :)